SlotMachine.hs

```diff
@@ -15,18 +15,19 @@ data SlotMachine = SlotMachine {
         pays :: [(String, Double)],
         spreads :: [(Char, Int, Double)],
         gambles :: [Int],
-        modes :: [Int]}
+        modes :: [Int]
+}
 
 -- Examples for testing ----------------------------------------------------------------------------
 slotMachine1 = SlotMachine {
-    symbols = zip "9JQKA7%$#!" [10, 8, 5, 5, 5, 3, 3, 2, 2, 1.0], -- [(Char, Double)]
+    symbols = zip "9JQKA7%$#!" [10, 8, 5, 5, 5, 3, 3, 2, 2, 1], -- [(Char, Double)]
```

Comment:
An integer literal has the polymorphic type `Num a => a` which will automatically resolve here to `Double` thanks to type inference.